test/e2e/cli/helm/helm.go

```diff
@@ -32,8 +35,137 @@ func InstallTestApplication(input model.UserInputs, user model.DBUser, port stri
 	Install(
 		"test-app-"+user.Spec.Username,
 		config.HELMTestAppPath,
-		"--set-string", fmt.Sprintf("connectionSecret=%s-%s-%s", input.ProjectName, input.Clusters[0].Spec.Name, user.Spec.Username),
+		"--set-string", fmt.Sprintf("connectionSecret=%s-%s-%s", input.Project.GetProjectName(), input.Clusters[0].Spec.Name, user.Spec.Username),
 		"--set-string", fmt.Sprintf("nodePort=%s", port),
 		"-n", input.Namespace,
 	)
 }
+
+func InstallCRDToNamespace(input model.UserInputs) {
```

Comment:
[q] is there a difference between this func and the one on line 53? CRDs don't reference any namespaces so I guess the end result should  be the same?